### PR TITLE
OWNERS update: move SDN to aliases

### PR DESCRIPTION
**Pull request by squeed**
There are so many SDN-related OWNERS files. Let's move them to aliases.

### CODE REVIEW

**Comment by squeed** [comment]
Note: as far as I can tell, none of the "DO NOT EDIT" files are actually auto-generated...

**Comment by squeed** [comment]
I assume the bot just doesn't understand owners aliases. The owners files seem fine.

**Comment by squeed** [comment]
@bear-redhat would you mind approving this?

**Review by droslean — APPROVED** [review]

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -182,3 +182,26 @@ aliases:
   - deepsm007
   - r4f4
   - jaypoulz
+sdn-approvers:
```
Comment (droslean) [inline OWNERS_ALIASES:185]: bad indentation :)

**Pull request merged**